### ISSUE TITLE
Client Allocation Request for: Hunan jinyiyun Storage Technology Co.

### ISSUE THREAD

**Issue by huangqian2021** [filecoin-project/filecoin-plus-client-onboarding#1327]
---

# Client Allocation Questions 

## Core Information
- Name: Hunan jinyiyun Storage Technology Co.
- Website / Social Media: 金逸云储（微信公众号）
- Region: Greater China
- DataCap Requested: 100TiB
- Addresses to be Notarized: f3wtdob5ymm5qy3t6w2ggk3bb4niq25e3zaolvzhrkzc2fkphrrvrsqau74qvhwncghqoutg6lztjqyrsnlysa
- Notary Requested: steven004

@huangqian2021 Please subscribe to notifications for this Issue to be aware of updates. Notaries may request additional information on the Issue.

**Comment by steven004** [comment]
Hey @huangqian2021 , thanks for your request. Can you kindly add more information / context here on:

1. yourself/ your organization (with links to official website/social media/news mentions etc)
2. Your history working on IPFS/Filecoin (notable contributions to the community, type of engagement shown over time etc)
3. Use case: type/source/quantity/diversity/scarcity of the data you plan to store on Filecoin
4. Miners you intend to allocate DataCap with allocation proportion(with redundancy/location/specific requirement of storage miners), if you are working with a service provider to store your data to Filecoin network, please name the provider.
5. How could people retrieve the data you store in Filecoin Network, do you have a tool/app for it?
6. What's your next plan for the data you plan to store in Filecoin Network? For-profit or not?
7. Would you be OK in posting deal transactions here?

Thanks

**Comment by huangqian2021** [comment]
Hi Steven, 




here is the information about us:




1.Hunan jinyiyunchu Technology Co. Ltd. established in 2013, is a technology enterprise focusing on distributed data storage and ecological application, and has established 18 branches and agents in various places.

WeChat official account：金逸云储.




2. We get to know about Filecoin when We have learned about Filecoin from May 2020 until now. At that time, we was noticed about the information of IPFS which published on social media, and paid attetion this aspect and began to focus on understanding for the technology.

        Projects that we participated :

         Space Race 1.

         Space Race 2,




         Published articles as below:

         Filecoin zk-SNARKs: Filecoin network has become the largest zero-knowledge proof network

https://mp.weixin.qq.com/s__biz=MzIwNTQ0NzkyOA==&mid=2247490163&idx=1&sn=008094ba53725685fc99208450daaa59&chksm=9731ede4a04664f231aa5cb4fbbdde70a4da802a98dzh3e7e308201330815




         The Autumn of Filecoin: Upstream of Value

https://mp.weixin.qq.com/s__biz=MzIwNTQ0NzkyOA==&mid=2247490422&idx=2&sn=15730ea89f0eede2b7ec25fa3b6c5227&chksm=9731ece1a04665f79bbb52631416a7292cc1a8986blang21950c8308eccCN620d=9731ece1a04665f79bbc8308bCN291686ecc




          Protocol Labs and Energy Web completed the first demonstration of Filecoin's carbon reduction open source solution

https://mp.weixin.qq.com/s__biz=MzIwNTQ0NzkyOA==&mid=2247490741&idx=5&sn=e9dff259edfdf63a1692d8dbb23a6ddb&chksm=9731eb22a0466234ff5c472491e18f903567db6f16a45e18f90356adb6db6f5c472491e18f903567db6f5c472491e18ddb&chksm=9731eb22a0466234ff5c472490




          How does the combination of Filecoin and IPFS realize decentralized storage

https://mp.weixin.qq.com/s__biz=MzIwNTQ0NzkyOA==&mid=2247490972&idx=2&sn=de327eabfb3547b7d69dd4cc69e15904&chksm=9731ea0ba046631dde1ba185d2rdlanguage=9731ea0ba046631dde1ba185e1014308CN554e1ba18429a1d2f30




3.Our data source mainly comes from Picture (jpg) data, there are about 200T total data. The data can be made public and retrieved by the hash value of the picture. As for the follow-up plan, we have other customers who can communicate to provide service. 




4.  Number of miners: about 10, depending on the situation

-f0218407

-f01038199

-f0441525

-f01174943

-f01174966

-f01213126

-f01213185

-f01271229

-f0241536

-f01384139 

Storage ratio of each miner: 10%




5. Our technical department has not developed this kind of APP for the time being, but is planning this kind of tool.




6. Our next plan is to storage more data in Filecoin Network and expand the scale, and we are not for- profit.




7.Yes!







Thank you so much!  

If you have any question ,please feel free to contact me.




Sophie huang

2021.12.27
















At 2021-12-27 10:09:04, "Steven Li" ***@***.***> wrote:

Hey @huangqian2021 , thanks for your request. Can you kindly add more information / context here on:

yourself/ your organization (with links to official website/social media/news mentions etc)
Your history working on IPFS/Filecoin (notable contributions to the community, type of engagement shown over time etc)
Use case: type/source/quantity/diversity/scarcity of the data you plan to store on Filecoin
Miners you intend to allocate DataCap with allocation proportion(with redundancy/location/specific requirement of storage miners), if you are working with a service provider to store your data to Filecoin network, please name the provider.
How could people retrieve the data you store in Filecoin Network, do you have a tool/app for it?
What's your next plan for the data you plan to store in Filecoin Network? For-profit or not?
Would you be OK in posting deal transactions here?

Thanks

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by steven004** [comment]
Hey @huangqian2021 , all the links in your answers can not be accessed, they always return 404, please help check and update.

**Comment by huangqian2021** [comment]
Hi Steve，


I have updated the links for your reference.



 Filecoin zk-SNARKs: Filecoin network has become the largest zero-knowledge proof network

https://mp.weixin.qq.com/s?__biz=MzIwNTQ0NzkyOA==&mid=2247490163&idx=1&sn=008094ba53725685fc99208450daaa59&chksm=9731ede4a04664f231aa5cb4fbbdde70a4da802a98d7543e7e1274ee7e134bb1201373cc1e55&token=308620900&lang=zh_CN#rd




The Autumn of Filecoin: Upstream of Value

https://mp.weixin.qq.com/s?__biz=MzIwNTQ0NzkyOA==&mid=2247490422&idx=2&sn=15730ea89f0eede2b7ec25fa3b6c5227&chksm=9731ece1a04665f79bb52631416a7292cc1a8986b950c8e938291e42b13b418d21686d6cc9ae&token=308620900&lang=zh_CN#rd




Protocol Labs and Energy Web completed the first demonstration of Filecoin's carbon reduction open source solution

https://mp.weixin.qq.com/s?__biz=MzIwNTQ0NzkyOA==&mid=2247490741&idx=5&sn=e9dff259edfdf63a1692d8dbb23a6ddb&chksm=9731eb22a0466234ff5c472491e18f90356adb6f16a45a55db410ee1979f7dd70e0f2082aae3&token=308620900&lang=zh_CN#rd




How does the combination of Filecoin and IPFS realize decentralized storage

https://mp.weixin.qq.com/s?__biz=MzIwNTQ0NzkyOA==&mid=2247490972&idx=2&sn=de327eabfb3547b7d69dd4cc69e15904&chksm=9731ea0ba046631dde1ba185429a1d2f3045006a1370fe1014acdc0e93acc554f5f2f029d9a9&token=308620900&lang=zh_CN#rd







Sophie huang 
2021.12.28






At 2021-12-28 15:21:20, "Steven Li" ***@***.***> wrote:

Hey @huangqian2021 , all the links in your answers can not be accessed, they always return 404, please help check and update.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by steven004** [comment]
Hey @huangqian2021 , thanks you for providing the information. I will grant you 5TiB as a start, you could upload the deal list before it is used up for verification. Once the verification is done, you can ask more.

**Comment by steven004** [comment]
Hey @huangqian2021 , Your Datacap Allocation Request has been approved

**Message sent to Filecoin Network**
> [bafy2bzaceaqo3rpvxeeg4ywtzntkudhh2t4a6xmspuayf3pnjhdiwznbd7bbe](https://filscan.io/tipset/message-detail?cid=bafy2bzaceaqo3rpvxeeg4ywtzntkudhh2t4a6xmspuayf3pnjhdiwznbd7bbe)

**Address**
> [f3wtdob5ymm5qy3t6w2ggk3bb4niq25e3zaolvzhrkzc2fkphrrvrsqau74qvhwncghqoutg6lztjqyrsnlysa](https://filscan.io/address/general?address=f3wtdob5ymm5qy3t6w2ggk3bb4niq25e3zaolvzhrkzc2fkphrrvrsqau74qvhwncghqoutg6lztjqyrsnlysa)

**Datacap Allocated**
> 5.0TiB
